iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/ThreeWindingsTransformerImpl.java

```diff
@@ -102,6 +119,11 @@ public String getMessageHeader() {
             return getTypeDescription() + " '" + transformer.getId() + "': ";
         }
 
+        protected String getLegAttribute() {
+            return "leg" + getLegNumber();
+        }
```

Comment:
To be discussed: maybe `getLegAttribute` is sufficient event the trace (and toString) are a little bit different